### PR TITLE
API: Fix importing plugin dashboards

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
#21350 introduced a bug regarding import of plugin dashboards. This should fix this and add custom validation if not importing plugin dashboard and dashboard property is missing.

image

**Which issue(s) this PR fixes**:
Ref #21350

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/api/plugins.go [hunk 1]
```diff
@@ -179,6 +179,10 @@ func GetPluginMarkdown(c *m.ReqContext) Response {
 }
 
 func ImportDashboard(c *m.ReqContext, apiCmd dtos.ImportDashboardCommand) Response {
+	if apiCmd.PluginId == "" && apiCmd.Dashboard == nil {
+		return Error(422, "dashboard must be set", nil)
```
Comment (aknuds1) [inline pkg/api/plugins.go]: AFAICT, our convention is to start HTTP error messages with a capital letter (like a sentence).
```suggestion
		return Error(422, "Dashboard must be set", nil)
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
So if I understand this PR correctly, dashboards only need to be provided if plugin ID is not?

**Comment by marefr** [comment]
> So if I understand this PR correctly, dashboards only need to be provided if plugin ID is not?

Yes

**Review by aknuds1 — APPROVED** [review]
LGTM!

**Pull request merged**